Ark.Tools.Outbox.SqlServer/RECDataContextFactory.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Ark.Tools.Core;
+using Ark.Tools.Sql;
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Ark.Tools.Outbox.SqlServer
+{
+    public class RECDataContextFactory : IContextFactory<IRECDataContext>, IAsyncDisposable
+    {
+        private readonly IRECDataContextConfig _contextConfig;
+        private readonly IDbConnectionManager _dbConnectionManager;
+        private DbConnection? _dbConnection;
```

Comment:
this makes no sense... this is a Factory, should not own any connection otherwise is required to be AsyncDisposable which is not supposed to be.

Is each Context instance that owns the Transaction.